front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -862,10 +862,10 @@ export function retrievalMetaPrompt() {
 
 export function retrievalMetaPromptMutiActions() {
   return (
-    "Focus on being factual and accurate. When data is retrieved from sources, " +
-    "use the markdown directive :cite[REFERENCE] to cite documents (eg :cite[XX] or :cite[XX,XX] but not :cite[XX][XX]). " +
-    "Ensure citations are placed as close as possible to the related information. " +
-    "If data retrieval isn't applicable, maintain clarity and precision in your statements."
+    "To cite documents retrieved with a 2-letter REFERENCE, " +
```

Comment:
This is injected as soon as there is at least one retrieval actions. So made it a bit more adapted to that new situation